### PR TITLE
Fix: Fixes lint errors discovered after the linting configuration was implemented

### PR DESCRIPTION
**Pull request by Akpjunior94**
**This pull request makes the following changes**
* Fixes lint error fixes/#56

**General checklist**
- [X] File or folder now contains changes as specified in the issue i worked on
- [x] I have linked the issue I worked on to this pull request submitted by me

**Testing checklist**

Replace this text with the testing checklist from the issue this pull request fixes.

Ping @code-collabo/web-app

### CODE REVIEW

**Review by Ifycode — CHANGES_REQUESTED** [review]
**_Review comment 1_**
- This PR is linked to the wrong issue. Link it to the ES lint issue.

**Comment by Akpjunior94** [comment]
> **_Review comment 1_**
> 
> * This PR is linked to the wrong issue. Link it to the ES lint issue.

Oh. my bad. done. please check again

File: client/src/apps/code-collabo/helpers/appInfo.ts [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
+}
+
+interface AppInfo {
+  id: number;
+  name: string;
+  pages: {
+    [key: number]: Page;
+  };
+}
+
```
Comment (Ifycode) [inline client/src/apps/code-collabo/helpers/appInfo.ts:3]: **_Review comment 2_**

These interfaces appear to be present in 3 different files @Akpjunior94 
Better to be inside one file and imported everywhere. The file you create for the interface should be in the shared folder's helper folder.

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: client/src/apps/code-collabo/helpers/appInfo.ts (hunk 1 above)
Comment (Akpjunior94) [reply client/src/apps/code-collabo/helpers/appInfo.ts:3]: Noted will update that as well

Comment (Akpjunior94) [reply client/src/apps/code-collabo/helpers/appInfo.ts:3]: Done @Ifycode please check.

**Review by Ifycode — APPROVED** [review]

**Pull request merged**